### PR TITLE
Translation INSTALL.md and MANUAL.md into Russian

### PR DESCRIPTION
**Pull request by sadykhzadeh**
## Type of changes

- [ ] Bug fix (changes which fix an issue)
- [ ] New feature (changes which add functionality)
- [ ] Documentation (changes which modify the documentation only)
- [ ] Style change (changes which do not affect the meaning of the code: code formatting, etc.)
- [ ] Refactor (changes which affect the meaning of the code but neither fix a bug nor add a feature)
- [ ] Performance improve (changes which improve performance)
- [ ] Test (changes which add tests)
- [ ] Build (Changes that affect the build system or external dependencies)
- [ ] CI (changes to CI configuration files and scripts)
- [ ] Chore (changes which do not belong to any type above)
- [ ] Revert (revert previous changes)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/cpeditor/cpeditor/blob/master/CONTRIBUTING.md) document.
- [ ] I have tested these changes locally, and this fixes the bug/the new feature behaves as the expectation.
- [ ] I have used clang-format-9 and `.clang-format` file in the root directory to format my codes.
- [ ] The settings file in the old version can be used in the new version after this change.
- [ ] These changes only fix a single bug/introduces a single feature. (Otherwise, open multiple Pull Requests instead, unless these bugs/features are closely related.)
- [ ] The commit messages are clear and detailed. (Otherwise, use `git reset` and commit again, or use `git rebase -i` and `git commit --amend` to modify the commit messages.)
- [ ] These changes don't remove an existing feature. (Otherwise, add an option to disable this feature instead, unless it's necessary to remove this feature.)
- [ ] I have documented these changes in [CHANGELOG.md](https://github.com/cpeditor/cpeditor/blob/master/doc/CHANGELOG.md), or these changes are not notable.

### CODE REVIEW

**Comment by ouuan** [comment]
Can you also translate [CONTRIBUTING.md](https://github.com/cpeditor/cpeditor/blob/master/CONTRIBUTING.md) and [README.md](https://github.com/cpeditor/cpeditor/blob/master/README.md)?

File: doc/INSTALL_ru-RU.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+1. [Установка из бинарных файлов](#Install-from-binaries)
+
+   Это самый простой способ, который рекомендуется для большинства пользователей.
+
+2. [Сборка из исходного кода](#Build-from-source)
+
+   Таким образом, вы можете получить самые последние неизданные функции и подготовиться к участию. Это требует дополнительных шагов, и это единственный выбор, если ваша ОС официально не поддерживается.
+
+3. [Использовать Артефакты](#Использовать-Артефакты)
```
Comment (ouuan) [inline doc/INSTALL_ru-RU.md:11]: I don't think "Artifacts" should be translated, it's like a proper noun on GitHub.

File: doc/INSTALL_ru-RU.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+   Этот способ рекомендуется для пользователей Arch Linux. Вы можете легко построить из исходного кода и получить новейшие функции. Более того, поскольку приложение AppImage построено на Ubuntu, вы можете использовать только тему fusion, но если она построена на Arch Linux с KDE, вы можете использовать и другие темы, такие как Breeze.
+
+## Установить из бинарных файлов
+
+Выберите версию в разделе [Releases](https://github.com/cpeditor/cpeditor/releases), затем загрузите свой двоичный файл.
+
+### Windows
+
+Загрузите `setup.exe` в активе и установите его. Также может потребоваться установить [распространяемый пакет Microsoft Visual C++ для Visual Studio 2015, 2017 и 2019](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads).
```
Comment (ouuan) [inline doc/INSTALL_ru-RU.md:25]:
```suggestion
Загрузите `setup.exe` в активе и установите его. Также может потребоваться установить [распространяемый компонент Microsoft Visual C++ для Visual Studio 2015, 2017 и 2019](https://support.microsoft.com/ru-ru/help/2977003/the-latest-supported-visual-c-downloads).
```

The new translation is copied from the official website (i.e. the link).

File: doc/INSTALL_ru-RU.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+		cmake --build . --config Release
+		```
+
+В Linux вы получите `build/cpeditor`.
+
+На Mac вы получите `build/cpeditor.app`.
+
+В Windows вы получите `build/cpeditor.exe`, или `build/Release/cpeditor.exe`. Возможно, вам потребуется собрать необходимые библиотеки DLL. Если вы установили редактор CP с помощью `setup.exe`, вы можете скопировать библиотеки DLL, из которых установлен редактор CP (или скопировать исполняемый файл в путь установки).
+
+## Использовать Артефакты
```
Comment (ouuan) [inline doc/INSTALL_ru-RU.md:79]: "Artifacts" shouldn't be translated. Remember to change the following paragraphs as well.

File: doc/INSTALL_ru-RU.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+Если вы хотите использовать последнюю версию (даже более позднюю, чем бета-версия), но не хотите строить из исходного кода, Вы можете загрузить артефакты из [действий GitHub](https://github.com/cpeditor/cpeditor/actions).
```
Comment (ouuan) [inline doc/INSTALL_ru-RU.md:81]:
```suggestion
Если вы хотите использовать последнюю версию (даже более позднюю, чем бета-версия), но не хотите строить из исходного кода, Вы можете загрузить артефакты из [GitHub Actions] (https://github.com/cpeditor/cpeditor/actions).
```

File: doc/INSTALL_ru-RU.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+Так же, как установка других пакетов из AUR, вам нужен помощник AUR, если у вас его нет, вы можете установить yay by `sudo pacman-S yay`.
+
+Затем запустите `yay-S cpeditor-git`, выберите параметры по умолчанию и дождитесь завершения сборки.
+
+После установки, вы сможете запустить редактор CP с помощью "cpeditor" в вашем терминале.
+
+# Начало Работы
+
+Возможно, вы захотите [Начать работу](MANUAL.md#Get-Started) после установки.
```
Comment (ouuan) [inline doc/INSTALL_ru-RU.md:101]:
```suggestion
Возможно, вы захотите [Начать работу](MANUAL_ru-RU.md#Начало-Работы) после установки.
```

File: doc/INSTALL_ru-RU.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
+# Выберите свой способ установки
+
+1. [Установка из бинарных файлов](#Install-from-binaries)
```
Comment (ouuan) [inline doc/INSTALL_ru-RU.md:3]:
```suggestion
1. [Установка из бинарных файлов](#Установить-из-бинарных-файлов)
```

File: doc/INSTALL_ru-RU.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+   Это самый простой способ, который рекомендуется для большинства пользователей.
+
+2. [Сборка из исходного кода](#Build-from-source)
```
Comment (ouuan) [inline doc/INSTALL_ru-RU.md:7]:
```suggestion
2. [Сборка из исходного кода](#Сборка-из-исходного-кода)
```

File: doc/INSTALL_ru-RU.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+   Таким образом, вы можете получить самые последние неизданные функции и подготовиться к участию. Это требует дополнительных шагов, и это единственный выбор, если ваша ОС официально не поддерживается.
+
+3. [Использовать Артефакты](#Использовать-Артефакты)
+
+    Таким образом, вы можете получить самые последние неизданные функции, не создавая их самостоятельно. Но не все операционные системы поддерживают это, и это доступно только для обновления, а не для первой установки.
+
+4. [Build from AUR](#Build-from-AUR)
```
Comment (ouuan) [inline doc/INSTALL_ru-RU.md:15]: Is this hard to translate? :thinking: I think it should be similar to "Build from source".

File: doc/INSTALL_ru-RU.md (hunk 1 above)
Comment (sadykhzadeh) [reply doc/INSTALL_ru-RU.md:11]: Okay, got it!

File: doc/INSTALL_ru-RU.md (hunk 8 above)
Comment (sadykhzadeh) [reply doc/INSTALL_ru-RU.md:15]: I haven't found any reliable Russian-language sources for ArchLinux users, so let's keep It that way for now.

**Comment by ouuan** [comment]
If you are willing to translate the contributing guidelines, please translate the latest version with f3d8c7c3b4206bc6f25990bd94b3c71e5163021b.

**Comment by coder3101** [comment]
> Can you also translate [CONTRIBUTING.md](https://github.com/cpeditor/cpeditor/blob/master/CONTRIBUTING.md) and [README.md](https://github.com/cpeditor/cpeditor/blob/master/README.md)?

Why README.md ? He should only add a link to Russian manual like you have added in README.

**Comment by ouuan** [comment]
> He should only add a link to Russian manual like you have added in README.

Like what? Add where? I don't understand.

**Comment by ouuan** [comment]
[This](https://github.com/cpeditor/cpeditor/blob/master/README_zh-CN.md) is the Chinese README, I don't think it's useless.

**Comment by sadykhzadeh** [comment]
> Can you also translate [CONTRIBUTING.md](https://github.com/cpeditor/cpeditor/blob/master/CONTRIBUTING.md) and [README.md](https://github.com/cpeditor/cpeditor/blob/master/README.md)?
Yes, sure.


> If you are willing to translate the contributing guidelines, please translate the latest version with [f3d8c7c](https://github.com/cpeditor/cpeditor/commit/f3d8c7c3b4206bc6f25990bd94b3c71e5163021b).
Of course.

**Comment by coder3101** [comment]
We frequently (in some) bring changes to README, CONTRIBUTING, MANUAL. How do you say we keep Russian translation up to date? Chinese translation are handled by @ouuan

**Comment by ouuan** [comment]
> We frequently (in some) bring changes to README, CONTRIBUTING, MANUAL. How do you say we keep Russian translation up to date? Chinese translation are handled by @ouuan

Do you mean a not-up-to-date translation is worse than nothing? And I believe @sadykhzadeh is willing to update the translation, though maybe not so immediately after updating other versions.

File: doc/INSTALL_ru-RU.md (hunk 1 above)
Comment (IZOBRETATEL777) [reply doc/INSTALL_ru-RU.md:11]: > I don't think "Artifacts" should be translated, it's like a proper noun on GitHub.

No, "Artifacts" can be translated. For example https://ru.werf.io/documentation/configuration/stapel_artifact.html

Comment (ouuan) [reply doc/INSTALL_ru-RU.md:11]: > No, "Artifacts" can be translated. For example https://ru.werf.io/documentation/configuration/stapel_artifact.html

OK

**Pull request closed without merging**